.gitignore

```diff
@@ -31,3 +31,6 @@ node_modules
 
 # Optional REPL history
 .node_repl_history
+
+# Personal
+.idea
```

Comment:
As it says, it's personal. It shouldn't be here. Use a global gitignore on your local environment instead. See the guide for more information: https://github.com/ideas/guides/tree/master/git#repository